### ISSUE TITLE
Not Able to Seek

### ISSUE THREAD

**Issue by tonkaskippers** [jellyfin/Swiftfin#703]
### Describe the bug

When watching a video, I cannot seek to a different portion of the video.  It lets me drag the playhead (sometimes wiggling back and forth quite a bit), but won't let me actually select that moment.  It just hides the controls and keeps playing like nothing changed.  Not much else to report.  It occurs every time.

### Application version

1.0—Released Dec 28, 2022

### Where did you install the app from?

App Store

### Device information

Apple TV 4th Gen

### OS version

tvOS 16.3.2 (20K672)

### Jellyfin server version

10.8.9

**Comment by Rukongai** [comment]
Experiencing this as well

**Comment by IreneKnapp** [comment]
I am also experiencing this. Please let me know if I can help in some way.

**Comment by P-St** [comment]
I also experienced this. As I use Direct Play only, I toggled "Settings->experimental->Force Direct Play" to "On" and now seeking works as expected (using v1.0.1).

**Comment by ryanlbates01** [comment]
I also have this issue.
Setting to direct play is fine but if I want to watch files that need transcoding this workaround doesn't work.


Apple TV 4th Gen (A1625), tvOS version: 17.4(21L227)
Server:
Version: 10.8.13
Operating System: Linux
Architecture: X64

**Comment by xMdb** [comment]
From having a look at some other issues, it seems like an issue with Apple TV 4th Gen. Newer devices don't have this issue.

As mentioned, a workaround is to use Settings -> Experimental -> Force Direct Play set to On.

I have the same issue. Since it's been around for a year, I have no hope and I am simply going to upgrade to a newer model, as I need to transcode most of my content :)

**Comment by ryanlbates01** [comment]
Hey xMdb 
Yeah I came to the same conclusion that it was a 4th gen issue.

I have force direct play enabled but it then doesn’t work for some of my content but I believe that’s coz the 4th gen doesn’t support the codecs of those files

So like you I am planning to upgrade my hardware to the latest 4K model as well.

I got the latest version of the app on my iPhone which seems like it works well, I’m hoping we will get an tvOS update soon as well

**Comment by xMdb** [comment]
> Hey xMdb Yeah I came to the same conclusion that it was a 4th gen issue.
> 
> I have force direct play enabled but it then doesn’t work for some of my content but I believe that’s coz the 4th gen doesn’t support the codecs of those files
> 
> So like you I am planning to upgrade my hardware to the latest 4K model as well.
> 
> I got the latest version of the app on my iPhone which seems like it works well, I’m hoping we will get an tvOS update soon as well

Yep. I think it's time to consume some 4K content on my 4K display. lol

I hope we get a tvOS update soon too. Although I did see in #1099 that it is still a bit further down the line as they are essentially separate projects. LePips and the team are doing some great work.

**Comment by jshwrner** [comment]
Same here. I've been using Infuse for the last year or so because the app for AppleTV 4th gen is honestly pretty bad. I ran into a few issues in which my AppleTVs (again, 4th gen) aren't able to play some higher end media files directly so I'm forced to transcode. I'm now in an unfortunate situation in which the AppleTV won't play those media files via Infuse (or direct) and the Jellyfin app is borderline unusable. I don't think it's reasonable to purchase a couple new AppleTVs to make up for the issues on 4th gen; especially since these aren't extremely old. Not being able to seek is killer, in my opinion.
